api/config/v1alpha1/envoygateway_types.go

```diff
@@ -67,10 +67,16 @@ type EnvoyGatewaySpec struct {
 	// +optional
 	RateLimit *RateLimit `json:"rateLimit,omitempty"`
 
-	// Extension defines an extension to register for the Envoy Gateway Control Plane.
+	// ExtensionManager defines an extension manager to register for the Envoy Gateway Control Plane.
 	//
 	// +optional
-	Extension *Extension `json:"extension,omitempty"`
+	ExtensionManager *ExtensionManager `json:"extensionManager,omitempty"`
```

Comment:
lgtm even though this is a breaking change for Envoy Gateway extensions. If possible we should draw extra attention to this in the release notes with a :warning: emoji or something.

Reply:
will do, adding the release notes label to ensure we handle ^